### PR TITLE
Testing accept comment feature on 4.13

### PR DESCRIPTION
**Pull request by jeana-redhat**
This is a test PR. More info [here](https://docs.google.com/document/d/1Gm7SRwD4Q_pJe5QSDNdG6Y1bIReD-W3OO4adL9PoJ3s/edit).

**Conditions**
Base branch: `enterprise-4.13`
Origin updated: Yes
Checkout against: `upstream/enterprise-4.13`

### CODE REVIEW

File: machine_management/index-test.adoc [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+====
+
+In bibendum tristique mauris, in tincidunt lectus sagittis et. Fusce egestas vehicula purus eget fermentum. Donec elit risus, dapibus non vulputate sed, ultrices vitae sapien. Sed ac sapien at sapien luctus efficitur ac sit amet sapien. Nulla imperdiet tempus aliquet. Pellentesque finibus, lectus sed suscipit vestibulum, nisl tellus laoreet elit, eget scelerisque mi lacus at mi. Nullam ut arcu leo.
+
+Integer lacinia sagittis quam eu euismod. Nullam aliquam, odio hendrerit bibendum sodales, tortor tortor ultrices ex, at dictum libero sem eget tortor. Nulla facilisi. Nunc vel laoreet ipsum. Ut at enim in lorem commodo semper eget nec mi. Nullam in malesuada sapien. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Pellentesque pretium sit amet nisi in varius. Suspendisse potenti. Duis vel lacus sem. Sed pulvinar, leo quis sollicitudin ullamcorper, dui lacus gravida enim, at vestibulum lorem nisi at diam. Praesent faucibus porttitor nisi, at fringilla libero sagittis quis. Etiam eleifend erat quis auctor rhoncus. Quisque porttitor nunc in mollis commodo.
```
Comment (nmuller66) [inline machine_management/index-test.adoc:21]:
```suggestion
Integer lacinia sagittis quam eu euismod. Nullam aliquam, odio hendrerit bibendum sodales, tortor tortor ultrices ex, at dictum libero sem eget tortor. Nulla facilisi. Nunc vel laoreet ipsum. Ut at enim in lorem commodo semper eget nec mi. Nullam in malesuada sapien. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Pellentesque pretium sit amet nisi in varius. Suspendisse potenti. Duis vel lacus sem. Sed pulvinar, leo quis sollicitudin ullamcorper, dui lacus gravida enim, at vestibulum lorem nisi at diam. Praesent faucibus porttitor nisi, at fringilla libero sagittis quis. This is some first class nonsense!
```

File: machine_management/index-test.adoc [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+[NOTE]
+====
+Nam nec dolor ut orci ultrices volutpat eu sit amet urna. In diam quam, aliquam ut tristique id, laoreet id tortor. Aliquam convallis ac lorem varius elementum. Integer auctor velit pretium, molestie turpis et, ornare arcu. Nam non orci at enim fringilla congue eget nec tortor. Duis tempor finibus felis quis bibendum. Vestibulum a metus aliquam, sollicitudin felis id, sagittis ante.
+====
+
+In bibendum tristique mauris, in tincidunt lectus sagittis et. Fusce egestas vehicula purus eget fermentum. Donec elit risus, dapibus non vulputate sed, ultrices vitae sapien. Sed ac sapien at sapien luctus efficitur ac sit amet sapien. Nulla imperdiet tempus aliquet. Pellentesque finibus, lectus sed suscipit vestibulum, nisl tellus laoreet elit, eget scelerisque mi lacus at mi. Nullam ut arcu leo.
```
Comment (maxwelldb) [inline machine_management/index-test.adoc:19]:
```suggestion
Willow catkins beat at the curtains,
Under sweltering spring clouds.
Screens of tortoise-shell 
And dazzling clothes.
```

File: machine_management/index-test.adoc [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+. Etiam maximus tellus a libero aliquam, vel eleifend dui ultrices.
++
+[source,terminal]
+----
+Nunc dapibus facilisis elit, quis sodales tortor rutrum dapibus.
+----
+
+[IMPORTANT]
+====
+Suspendisse quis dictum ipsum.
```
Comment (ShaunaDiaz) [inline machine_management/index-test.adoc:38]:
```suggestion
I have eaten
the plums
that were in
the icebox

and which
you were probably
saving
for breakfast

Forgive me
they were delicious
so sweet
and so cold
```

File: machine_management/index-test.adoc [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+In id neque et ligula egestas interdum.
```
Comment (ShaunaDiaz) [inline machine_management/index-test.adoc:39]:
```suggestion
```

File: machine_management/index-test.adoc [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+Morbi convallis vitae velit eget pulvinar.
```
Comment (ShaunaDiaz) [inline machine_management/index-test.adoc:40]:
```suggestion
```

**Comment by jeana-redhat** [comment]
I messed this one up! Learning...

**Pull request closed without merging**